app/src/main/res/values/strings.xml

```diff
@@ -1,10 +1,11 @@
 <resources>
-    <string name="common_ok">OK</string>
-    <string name="common_yes">Yes</string>
-    <string name="common_no">No</string>
-    <string name="common_cancel">Cancel</string>
-    <string name="common_delete">Delete</string>
-    <string name="common_remove">Remove</string>
+    <!-- Common strings, uncomment if you want to use them -->
+    <!--<string name="common_ok">OK</string>-->
+    <!--<string name="common_yes">Yes</string>-->
+    <!--<string name="common_no">No</string>-->
+    <!--<string name="common_cancel">Cancel</string>-->
+    <!--<string name="common_delete">Delete</string>-->
+    <!--<string name="common_remove">Remove</string>-->
 
```

Comment:
Opinion: I think we should probably just remove these. It's pretty explicit that we're using strings.xml for common or app specific copy and words. When developers clone this project to kickstart a new product order, I'm assuming these will be deleted more than they are uncommented. If we want to keep them, I'd recommend making common_strings.xml and having them commented out there.